label_studio_converter/brush.py

```diff
@@ -143,4 +147,171 @@ def convert_task_dir(items, out_dir, out_format='numpy'):
+            if length_reeks <= 8:
+                # Starting with a 1 indicates that we have started a series
+                out_str += '1'
+
+                # index in rle size arr
+                out_str += '00'
+
+                # length of array to bits
+                out_str += f'{length_reeks - 1:03b}'
+
+                # get values
+                value = val_arr[0]
+                out_str += f'{value:08b}'
+
+            # rle size = 4
+            elif 8 < length_reeks <= 16:
+                # Starting with a 1 indicates that we have started a series
+                out_str += '1'
+                out_str += '01'
+
+                # length of array to bits
+                out_str += f'{length_reeks - 1:04b}'
+
+                # Get values
+                value = val_arr[0]
+                out_str += f'{value:08b}'
+
+            # rle size = 8
+            elif 16 < length_reeks <= 256:
+                # Starting with a 1 indicates that we have started a series
+                out_str += '1'
+
+                out_str += '10'
+
+                # length of array to bits
+                out_str += f'{length_reeks - 1:08b}'
+
+                # Get values
+                value = val_arr[0]
+                out_str += f'{value:08b}'
+
+            # rle size = 16 or longer
+            else:
+
+                length_temp = length_reeks
+                while length_temp > 2 ** 16:
+                    # Starting with a 1 indicates that we have started a series
+                    out_str += '1'
+
+                    out_str += '11'
+                    out_str += f'{2 ** 16 - 1:016b}'
+
+                    # add the value in 8 bit string
+                    value = val_arr[0]
+                    out_str += f'{value:08b}'
+                    length_temp -= 2 ** 16
+
+                # Starting with a 1 indicates that we have started a series
+                out_str += '1'
+
+                out_str += '11'
+                # length of array to bits
+                out_str += f'{length_temp - 1:016b}'
+
+                # get value
+                value = val_arr[0]
+                out_str += f'{value:08b}'
+
+    # make sure that we have an 8 fold lenght otherwise add 0's at the end
+    nzfill = 8 - len(base_str + out_str) % 8
+    total_str = base_str + out_str
+    total_str = total_str + nzfill * '0'
+
+    rle = bits2byte(total_str)
+
+    return rle
+
+
+def mask2rle(contours, contour_id, img_width, img_height):
+    """
+    :param contours:  list of contours
+    :type contours: list
+    :param contour_id: id of contour which you want to translate
+    :type contour_id: int
+    :param img_width: image shape width
+    :type img_width: int
+    :param img_height: image shape height
+    :type img_height: int
+    :return:
+    """
+
+    mask_im = np.zeros((img_width, img_height, 4))
+    mask_contours = cv2.drawContours(mask_im, contours, contour_id, color=(0, 255, 0, 100), thickness=-1)
```

Comment:
@bram2506 Where is cv2 imported? And I think we need to include it into requirements.txt